monai/transforms/croppad/array.py

```diff
@@ -282,15 +282,16 @@ def __init__(
         self.random_center = random_center
         self.random_size = random_size
         self._size: Optional[Sequence[int]] = None
-        self._slices: Optional[Tuple[slice, ...]] = None
+        self._slices: Optional[Sequence[slice]] = None
 
     def randomize(self, img_size) -> None:  # type: ignore # see issue #729
         self._size = fall_back_tuple(self.roi_size, img_size)
         if self.random_size:
             self._size = tuple((self.R.randint(low=self._size[i], high=img_size[i] + 1) for i in range(len(img_size))))
         if self.random_center:
             valid_size = get_valid_patch_size(img_size, self._size)
-            self._slices = (slice(None),) + get_random_patch(img_size, valid_size, self.R)
```

Comment:
Was there a reason to change this? Leaving it as tuple shouldn't be a problem and means you don't have to cast it to tuple when applied as indices.

Reply:
> Was there a reason to change this? Leaving it as tuple shouldn't be a problem and means you don't have to cast it to tuple when applied as indices.

Hi @ericspod , because we changed the typehint of self._slices to `Sequence[slice]`, it can't work with tuple.
Do you mean to set its type to `Tuple[slice, ...]`?

Reply:
It should stay as Tuple then perhaps since it isn't an argument, though I thought Sequence would work with tuple or list values.

Reply:
Sure, updated in the latest commit.
Thanks.